src/TaskSerializer/DefaultTaskSerializer.ts

```diff
@@ -41,9 +43,11 @@ export interface DefaultTaskSerializerSymbols {
  */
 export const DEFAULT_SYMBOLS: DefaultTaskSerializerSymbols = {
     prioritySymbols: {
+        Highest: '🔝',
```

Comment:
Or we could go the other way, and change medium priority to use one of these ⏹️/⏺️/*️⃣, then without problem high could use 🔼 and highest could use ⏫️. What do you think?

Reply:
> Or we could go the other way, and change medium priority to use one of these ⏹️/⏺️/*️⃣, then without problem high could use 🔼 and highest could use ⏫️. What do you think?

😱 

If you change the meaning of an existing symbol, you will have to be confident that every single Tasks plugin user in the world will update all their uses of that symbol in all their notes in all their vaults!

Otherwise, their queries will change meaning...

Which is a roundabout way of saying, while my earlier comments about not changing undocumented behaviour remain valid, changing documented behaviour of emoji is absolutely a no! 🤣 

https://medium.com/ula-engineering/ensuring-backwards-compatibility-in-your-applications-5107591e9968

Reply:
ok, you are right, many people surely might be using symbols e.g. in templates, I didn't think it through.

*️⃣ for highest might be the best idea. ⏏️/⬆️/⤴️/🆙 are also alternatives, they are in the same style and have a "something high" reasoning behind them. Although ⏏️/⬆️/⤴️ doesn't really exactly look higher than ⏫️ :s. Not sure about 🆙, what do you think?

Reply:
> ok, you are right, many people surely might be using symbols e.g. in templates, I didn't think it through.

Thank you - and good point about templates too.

> *️⃣ for highest might be the best idea. ⏏️/⬆️/⤴️/🆙 are also alternatives, they are in the same style and have a "something high" reasoning behind them. Although ⏏️/⬆️/⤴️ doesn't really exactly look higher than ⏫️ :s. Not sure about 🆙, what do you think?

I had very similar thoughts. *️⃣ for highest was, for me, the only one that felt unambiguously higher than the alternatives.

I thought about 🆙 but had the feeling to more conveyed direction than importance.

Reply:
ok, I will fix the icon to *️⃣

Reply:
we have a problem. *️⃣ somehow isn't a one sign, but two signs one after the other, which then are displayed as one
https://emojipedia.org/keycap-asterisk/
at the moment I don't know how to handle that from the perspective of the code 🫤, maybe you know? As the DefaultTaskSerializer stops working correctly with it.
For the time being I used 🆙, as it is the only one making sense that is working correctly.

Reply:
ok, after some fight it seems I fixed that, pls check out in the commit:
https://github.com/obsidian-tasks-group/obsidian-tasks/pull/1943/commits/42d3dcbca0070c53da0229797095ae1867e977ad

Reply:
Nope, it's not 100% working:
<img width="286" alt="image" src="https://github.com/obsidian-tasks-group/obsidian-tasks/assets/6050673/9c8a5ade-43b7-44c8-9757-e45e88e4f900">
should we still fight with this icon (any suggestions what's wrong?) or do we go with another one?
Another problem with *️⃣ is that you cannot remove it with one backspace, as it is, as I said, more than one letter

Reply:
Thank you very much for the updates. I will get to this as soon as I can, hopefully early next week (7 days away).

Reply:
I don't have any other ideas for emojis unfortunately. Any more research you can do for ideas would be greatly appreciated.

It would be good to find something more logical and recognisable than UP.

Reply:
Well, I'll think about it.
For now only UP seems reasonable.

Another approach would be to stray a bit from the consistency and go with one of these: ❗️⚡️🆘. SOS has another color, but on the other hand, it's still a box, and it makes sense for the highest priority to change color to red. But I'm not sure SOS message really makes sense, it's more like the task needs some help. Exclamation mark is obvious, thunder is okaish too.

Reply:
Thanks ever so much for looking. 

I was going to say the exclamation mark looked very good but that triangle is even better ❤️

Reply:
Great, I changed to 🔺 :).

Reply:
> Great, I changed to 🔺 :).

Thanks - I like it! Resolving conversation.